homeassistant/components/linky/config_flow.py

```diff
@@ -0,0 +1,108 @@
+"""Config flow to configure the Linky integration."""
+import logging
+
+import voluptuous as vol
+from pylinky.client import LinkyClient, PyLinkyError
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_PASSWORD, CONF_TIMEOUT, CONF_USERNAME
+from homeassistant.core import callback
+
+from .const import DEFAULT_TIMEOUT, DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+_LOGGER.error("LINKY_CONFIG_FLOW")
+
+
+@config_entries.HANDLERS.register(DOMAIN)
+class LinkyFlowHandler(config_entries.ConfigFlow):
```

Comment:
Pass the domain as keyword argument instead of decorating:
https://developers.home-assistant.io/docs/en/next/config_entries_config_flow_handler.html#defining-your-config-flow